hail/python/hail/experimental/db.py

```diff
@@ -33,8 +33,8 @@ def from_json(doc: dict) -> 'DatasetVersion':
         -------
         :class:`.DatasetVersion`
         """
-        assert 'url' in doc, doc
-        assert 'version' in doc, doc
+        assert 'url' in doc
+        assert 'version' in doc
```

Comment:
In Python, the `, ...` provides more detail. For example:
```
In [1]: key = 'B'                                                                                                                                                                                                                                                               

In [2]: doc = {'A': 1}                                                                                                                                                                                                                                                          

In [3]: assert key in doc                                                                                                                                                                                                                                                       
Traceback (most recent call last):
  File "<ipython-input-3-c6a5ebd6a2ac>", line 1, in <module>
    assert key in doc
AssertionError


In [4]: assert key in doc, doc                                                                                                                                                                                                                                                  
Traceback (most recent call last):
  File "<ipython-input-4-28f2c139973f>", line 1, in <module>
    assert key in doc, doc
AssertionError: {'A': 1}

```

Let's keep them

Reply:
ah gotcha, just put them back in